[38;21m2025-01-03 05:27:13,355 :: DEBUG :: icloudpy.base :: base.py :: 240 :: Using session file /tmp/icloudpy/runner/quentintarantinohotmailfr.session[0m
[38;5;39m2025-01-03 05:27:13,356 :: INFO :: icloudpy.base :: base.py :: 247 :: Session file does not exist[0m
[38;21m2025-01-03 05:27:13,356 :: DEBUG :: icloudpy.base :: base.py :: 313 :: Authenticating as quentintarantino@hotmail.fr[0m
[38;21m2025-01-03 05:27:13,364 :: DEBUG :: icloudpy.base :: base.py :: 417 :: Authentication completed successfully[0m
[38;5;196m2025-01-03 05:27:13,366 :: ERROR :: root :: sync_drive.py :: 161 :: Unhandled file type - cannot unpack the package image/jpeg.[0m
